cppseminar/presentation/Pages/Admin/TestCase/Create.cshtml

```diff
@@ -2,7 +2,10 @@
 @model presentation.Pages.Admin.TestCase.CreateModel
 
 <h4>Create new test case</h4>
-<th:div asp-validation-summary="All" class="alert-danger p-2"></th:div>
+@if (!ViewData.ModelState.IsValid) {
```

Comment:
How is this relevant to paging?

Reply:
My bad, I added commit which only contains small changes that I was supposed to do together with paging. Such as hiding this error box and redirecting user to submission after creating it.